src/__tests__/validate-and-fix_test.js

```diff
@@ -45,6 +45,115 @@ describe("#validateAndFix", () => {
         expect(result).toBeTrue();
     });
 
+    it("should resolve false if the target of a tag does not exist", async () => {
```

Comment:
nit: I found `... where target file does not reference source file` from marker-parser_test.js to be more descriptive of this situation.